Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "vss-website"
-version = "0.1.0"
-authors = ["Timothy Yuen <timothy.yuen22@gmail.com"]
+version = "0.2.0"
+authors = ["Timothy Yuen <timothy.yuen22@gmail.com"] ["Neiro <neirosystem@aol.com>"]
```

Comment:
This will actually break parsing of the file. I believe the correct format is `["author name <author email", "other author name <other author email>"]`